fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -48,6 +48,18 @@ def self.ci?
       return false
     end
 
+    def self.unix?
+      !windows?
+    end
+
+    def self.windows?
+      (/cygwin|mswin|mingw|bccwin|wince|emx/ =~ RUBY_PLATFORM) != nil
```

Comment:
I see that this code is taken from an answer in http://stackoverflow.com/questions/170956/how-can-i-find-which-operating-system-my-ruby-program-is-running-on ... Please add a comment stating where it was taken from.